### PR TITLE
Add GTK as target template

### PR DESCRIPTION
**Pull request by mallibone**
As described in [issue](https://github.com/fsprojects/Fabulous/issues/259) this adds GTK as template to Fabulous.

* Add empty Template for GTK
* Add GTK to the dotnet template

### CODE REVIEW

**Comment by TimLariviere** [comment]
I've updated NuGet.exe and it fixed the Travis CI build fail.
Could you rebase your PR on the last commit on `master` please?

**Comment by mallibone** [comment]
Cool thanks for fixing the build 🙂 Updating `NuGet.exe` why didn't I think of that myself 🙈

File: templates/content/blank/NewApp.Gtk/NewApp.Gtk.fsproj [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  <Import Project="..\packages\Xamarin.Forms.XamarinFormsSdk\build\netstandard2.0\Xamarin.Forms.props" Condition="Exists('..\packages\Xamarin.Forms.XamarinFormsSdk\build\netstandard2.0\Xamarin.Forms.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{C6A6BA94-897C-48B5-8046-3DF33897BA94}</ProjectGuid>
+    <UseStandardResourceNames>true</UseStandardResourceNames>
+    <OutputType>WinExe</OutputType>
+    <RootNamespace>NewApp.Gtk</RootNamespace>
+    <AssemblyName>NewApp.Gtk</AssemblyName>
+    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
```
[the remark this answers is not in the document]
Comment (mallibone) [reply templates/content/blank/NewApp.Gtk/NewApp.Gtk.fsproj]: Currently Mono does not yet support 472. Though I think a push has been made. So perhaps in the near future one could bump the default .Net Version.

**Comment by TimLariviere** [comment]
I have tested it on a freshly installed Ubuntu Mate and it worked the first time :)
👍

File: templates/content/blank/NewApp.Gtk/NewApp.Gtk.fsproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (TimLariviere) [reply templates/content/blank/NewApp.Gtk/NewApp.Gtk.fsproj]: Ok, that's fine then. :)

File: templates/content/blank/NewApp.Gtk/Main.fs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿namespace NewApp.Gtk
+    module Main =
```
[the remark this answers is not in the document]
Comment (mallibone) [reply templates/content/blank/NewApp.Gtk/Main.fs]: Adjusted

File: templates/content/blank/NewApp.Gtk/packages.config [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Fabulous.Core" version="FabulousPkgsVersion" targetFramework="net471" />
```
[the remark this answers is not in the document]
Comment (mallibone) [reply templates/content/blank/NewApp.Gtk/packages.config]: Removed `targetFramework` as is done in other platform templates.

**Comment by mallibone** [comment]
Code fixes are now implemented. Also found an issue with the template under Windows and fixed it.

**Pull request merged**